### PR TITLE
Add left panel to main page(UI only)

### PR DESCRIPTION
**Pull request by powerseed**
(This PR only contains UI changes)
1. Used an array to keep all platforms selected by the user.
2. Add the selected platforms in to the query params.
3. When there is no mention displayed, show an instruction message as below:
image

### CODE REVIEW

**Review by grogsy — APPROVED** [review]
Checked it out, looking good!

File: client/src/pages/Main.js [hunk 1]
```diff
@@ -1,32 +1,202 @@
...
+        let temp = this.state.platformSelected;
+        temp.splice(index, 1);
+        await this.setState({
+          platformSelected: temp,
+        });
+      } else {
+        await this.setState({
+          platformSelected: [...this.state.platformSelected, value.target.name],
+        });
+      }
```
Comment (jt5301) [inline client/src/pages/Main.js]: The toggles are off by default, so does that mean that there should be no mentions that are displayed when the user logs in? just wondering if that might look strange if the right half is empty. Would it be enough to just add a message telling them to toggle on some feeds?

Comment (powerseed) [reply client/src/pages/Main.js]: > The toggles are off by default, so does that mean that there should be no mentions that are displayed when the user logs in? just wondering if that might look strange if the right half is empty. Would it be enough to just add a message telling them to toggle on some feeds?

You are right,  I think a message will be helpful to teach users how to use. I will make this change soon.

Comment (powerseed) [reply client/src/pages/Main.js]: image
Done. Please review it.

Comment (jt5301) [reply client/src/pages/Main.js]: That works! Thank you

**Review by jt5301 — APPROVED** [review]

**Pull request merged**